src/EditorFeatures/CSharpTest/Completion/CompletionProviders/Snippets/CSharpLSPSnippetTests.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Completion;
+using Microsoft.CodeAnalysis.CSharp.Completion.CompletionProviders.Snippets;
+using Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Completion.CompletionProviders.Snippets
+{
+    public class CSharpLSPSnippetTests : AbstractCSharpSnippetCompletionProviderTests
```

Comment:
this class is named weird.  should it be CSharpConsoleWriteLineSnippetTests?

Reply:
I'm just going to delete these tests. They were initially created to test the created LSP snippet string. They're not testing anything unique with the addition of the tests for the converter at this point.